subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -4583,8 +4583,7 @@ static inline int length_req_rsp_recv(struct ll_conn *conn, memq_link_t *link,
 			conn->llcp_length.tx_time = eff_tx_time;
 #endif /* CONFIG_BT_CTLR_PHY */
 
-			conn->llcp_length.ack = (conn->llcp_length.req -
-						 1);
+			conn->llcp_length.ack = (conn->llcp_length.req - 1);
```

Comment:
nitpick change request, remove parenthesis. 

To experiment with, use of git commit --amend and git push -f.

Reply:
Fixed by amend